projects/1.16/assets/redstone-control-2/rs_ctr2/lang/zh_cn.json

```
@@ -1,6 +1,10 @@
 {
   "itemGroup.rs_ctr2": "红石控制 2",
+  "rs_ctr2.subtitle.switch_flip": "已切换开关",
```

字幕格式错误。原版的字幕格式为`声源：音效`（其中声源有时不出现），如：
```
拉杆：拉动
从蜂蜜块滑下
```
这里需要与原版格式统一。下面数项同理。
（这里我给的实际上就是参照上面第一条的；不过估计这个开关不是“拉动”的？如果觉得这样翻译也不合适，可以自行更改。）
```suggestion
  "rs_ctr2.subtitle.switch_flip": "开关：拨动",
```

---

还真没意识到是音效……

---

subtitle 就是字幕。